fhir-persistence/src/main/java/com/ibm/fhir/persistence/FHIRPersistenceTransaction.java

```diff
@@ -25,44 +41,37 @@
     void begin() throws FHIRPersistenceException;
 
     /**
-     * Commit the current thread's transaction.
+     * End the current thread's transaction. If setRollbackOnly has been called,
+     * then roll back the transaction instead.
+     * This call only affects the current transaction if this object actually
+     * started the transaction (i.e. is the outermost instance of a transaction).
      * @throws Exception
      */
-    void commit() throws FHIRPersistenceException;
-
-
-    /**
-     * Roll back the current thread's transaction.
-     * @throws Exception
-     */
-    void rollback() throws FHIRPersistenceException;
+    void end() throws FHIRPersistenceException;
 
     /**
      * Modify the transaction associated with the current thread such that the only possible outcome of the transaction
      * is to roll back the transaction.
      * @throws FHIRPersistenceException
      */
     void setRollbackOnly() throws FHIRPersistenceException;
+    
 
     /**
-     * Enroll in an existing transaction.
-     *
-     * <p>Enrolling in an existing transaction is an alternative to beginning a new transaction. Calling this method
-     * gives implementations a chance to create necessary resources associated with a given unit of work when that
-     * unit of work is performed under an existing user-managed transaction.
-     *
+     * Commit the transaction (if we started it).
+     * Use {@link #end()} instead.
      * @throws FHIRPersistenceException
      */
-    void enroll() throws FHIRPersistenceException;
+    @Deprecated
+    default void commit() throws FHIRPersistenceException { end(); }
```

Comment:
Please update JDBCSearchNearTest or create a back log issue to update and remove commit

Reply:
Creating a backlog issue.